tailwind.config.js

```diff
@@ -17,9 +17,11 @@ module.exports = {
           light: "#F4F4F4",
         },
         khaki: "#c7d3d0",
+        darkGray: "#3F3E3E",
```

Comment:
I'd like to suggest the following, where darkGray is part of the gray scale. Or we could maybe replace 700 with "dark" to give us the `text-gray-dark` class.
```suggestion
        gray: {
          700: "#3f3e3e",
        },
```

Reply:
Great idea!